From aa2cbb707e44fbdc81f6d41ea02ea98c87c0c90f Mon Sep 17 00:00:00 2001
From: Caesar Wang <wxt@rock-chips.com>
Date: Fri, 10 Nov 2017 16:33:32 +0800
Subject: [PATCH] net: wireless: rockchip_wlan: avoid the unused the print

As the ap6212 used on kylin board, the kylin always got the error log
when we connected the AP. Says:
..
[   70.169440] CFG80211-ERROR) wl_cfg80211_get_station :
    wl_cfg80211_get_station: if_counters not supported ret=-23
[   70.184371] CFG80211-ERROR) wl_cfg80211_get_station :
    wl_cfg80211_get_station: if_counters not supported ret=-23
..

And the normal status, the ap6212 wifi always print the ssid and channel
message for output. Says:
..
[  125.172243] wl_run_escan: LEGACY_SCAN sync ID: 4, bssidx: 0
[  178.181897] wl_run_escan: LEGACY_SCAN sync ID: 5, bssidx: 0
[  241.192452] wl_run_escan: LEGACY_SCAN sync ID: 6, bssidx: 0
..

The AP6212 don't need these logs for default status.

Change-Id: I43455bcc277e380daf15ee91a1436cdc81a95039
Signed-off-by: Caesar Wang <wxt@rock-chips.com>
---
 drivers/net/wireless/rockchip_wlan/rkwifi/bcmdhd/Makefile     | 2 +-
 .../net/wireless/rockchip_wlan/rkwifi/bcmdhd/wl_cfg80211.c    | 4 +++-
 2 files changed, 4 insertions(+), 2 deletions(-)

diff --git a/drivers/net/wireless/rockchip_wlan/rkwifi/bcmdhd/Makefile b/drivers/net/wireless/rockchip_wlan/rkwifi/bcmdhd/Makefile
index 77d7fbcc5946..0dc7f6819a5a 100644
--- a/drivers/net/wireless/rockchip_wlan/rkwifi/bcmdhd/Makefile
+++ b/drivers/net/wireless/rockchip_wlan/rkwifi/bcmdhd/Makefile
@@ -18,7 +18,7 @@ DHDCFLAGS = -Wall -Wstrict-prototypes -Dlinux -DBCMDRIVER -DSDTEST       \
 	-DKEEP_ALIVE -DPKT_FILTER_SUPPORT -DPNO_SUPPORT -DDHDTCPACK_SUPPRESS  \
 	-DDHD_DONOT_FORWARD_BCMEVENT_AS_NETWORK_PKT -DRXFRAME_THREAD          \
 	-DTSQ_MULTIPLIER -DMFP -DWL_EXT_IAPSTA                                \
-	-DENABLE_INSMOD_NO_FW_LOAD                                            \
+	-DENABLE_INSMOD_NO_FW_LOAD -DDHD_UNSUPPORT_IF_CNTS                    \
 	-Idrivers/net/wireless/rockchip_wlan/rkwifi/bcmdhd \
 	-Idrivers/net/wireless/rockchip_wlan/rkwifi/bcmdhd/include
 
diff --git a/drivers/net/wireless/rockchip_wlan/rkwifi/bcmdhd/wl_cfg80211.c b/drivers/net/wireless/rockchip_wlan/rkwifi/bcmdhd/wl_cfg80211.c
index 88508d645dbb..e24608893f45 100644
--- a/drivers/net/wireless/rockchip_wlan/rkwifi/bcmdhd/wl_cfg80211.c
+++ b/drivers/net/wireless/rockchip_wlan/rkwifi/bcmdhd/wl_cfg80211.c
@@ -2789,7 +2789,9 @@ wl_run_escan(struct bcm_cfg80211 *cfg, struct net_device *ndev,
 
 		err = wldev_iovar_setbuf(ndev, "escan", params, params_size,
 			cfg->escan_ioctl_buf, WLC_IOCTL_MEDLEN, NULL);
-		printf("%s: LEGACY_SCAN sync ID: %d, bssidx: %d\n", __FUNCTION__, params->sync_id, bssidx);
+		WL_DBG(("LEGACY_SCAN sync ID: %d, bssidx: %d\n",
+			params->sync_id, bssidx));
+
 		if (unlikely(err)) {
 			if (err == BCME_EPERM)
 				/* Scan Not permitted at this point of time */
-- 
2.35.3

